simulators/portal-interop/src/constants.rs

```diff
@@ -8,9 +8,6 @@ pub const CONSTANTINOPLE_BLOCK_NUMBER: u64 = 7280000;
 pub const BYZANTIUM_BLOCK_NUMBER: u64 = 4370000;
 pub const HOMESTEAD_BLOCK_NUMBER: u64 = 1150000;
 
-// Tests will be automatically generated from the data in this file.
-// Test data is written in a yaml key/value format
-// For history tests, the header MUST be placed before its respective block body/receipt, to ensure that the data necessary for validation is available.
 // TESTS THAT USE THIS FILE
```

Comment:
Just to clarify, I think that allll of this comment should be moved over to the test file. Including the list of generated tests

Reply:
But the test data file will no longer be located in Portal-Hive in a week

Reply:
Ok I moved the "generated test cases"

Reply:
Ahhh, interesting. I guess what I'm trying to avoid with these comments is the case where somebody just adds content key/value pairs to the file without understanding the ramifications of doing so (i.e. a set of new tests will be generated & ran in portal-hive). But, the separation of repos does make this tricky. Though, I guess I still think it will be helpful to have these comments inside the new test-data repo. Or maybe moved to a `README` in the new repo...

Reply:
Sounds good